src/shared/components/mutationTable/column/RsidColumnFormatter.tsx

```diff
@@ -0,0 +1,99 @@
+import * as React from 'react';
+import {Circle} from "better-react-spinkit";
+import 'rc-tooltip/assets/bootstrap_white.css';
+import {Mutation} from "shared/api/generated/CBioPortalAPI";
+import {default as TableCellStatusIndicator, TableCellStatus} from "shared/components/TableCellStatus";
+import { VariantAnnotation } from 'shared/api/generated/GenomeNexusAPI';
+import GenomeNexusCache, { GenomeNexusCacheDataType } from "shared/cache/GenomeNexusCache";
+
+export default class RsidColumnFormatter {
+
+    public static renderFunction(data:Mutation[],
+                                 genomeNexusCache:GenomeNexusCache|undefined) {
+        const genomeNexusCacheData = RsidColumnFormatter.getGenomeNexusDataFromCache(data, genomeNexusCache);
+        return (
+            <div>
+                <span>{RsidColumnFormatter.getRsidViz(genomeNexusCacheData)}</span>
```

Comment:
do we really need the `<span>` around this component?

Reply:
Yes I think you are right, we don't need it. already removed it.

Reply:
@onursumer